ci: fixed ansible-lint errors and warnings

webarchitect609 opened the pull request:
- fixed: error "syntax-check 'src' is not a valid attribute for a Play";
- fixed: error "internal-error the role 'geerlingguy.repo-remi' was not
  found
  in ...";
- fixed: warning "[line-length] line too long (173 > 120 characters)".

geerlingguy commented:
Thanks! I'm holding off on touching anything more in my roles currently until a Galaxy bug gets worked out, so I'll wait until then to consider merging this.

geerlingguy commented:
Hmm... I might use this though (contrary to what I said in the issue you raised earlier...).

webarchitect609 commented:
@geerlingguy , could you please consider using `ansible-lint` again? [The Galaxy bug](https://github.com/ansible/galaxy/issues/2622#issuecomment-792808354) you were waiting for is now fixed. I've updated my PR and triple-checked everything, including [the latest ansible-lint docs](https://ansible-lint.readthedocs.io/en/latest/configuring.html#configuration-file) compliance. And finally, I've rebased & squashed everything in a single commit, so please, checkout my new PR #66
